configuration/persistence.md

```diff
@@ -11,59 +11,73 @@ openHAB can store data over time; this is known as persistence.
 
-Persistence Strategies are configured in a file named `<persistenceservice>.persist`, stored in `$OPENHAB_CONF/persistence` ("persistenceservice" is replaced by the name of your add-on (e.g. `rrd4j.persist`)).
+Persistence Strategies are configured in a file named `<persistenceservice>.persist`, stored in `$OPENHAB_CONF/persistence` (Replace "persistenceservice" with the name of your persistence add-on (e.g. `rrd4j.persist`)).
 
-Strategies may be used to persist an Item state when some event has occurred (e.g. an Item state has been updated or changed), on a time schedule, or at a specific time of day (e.g. through a [cron expression](http://www.quartz-scheduler.org/documentation/quartz-2.1.x/tutorials/crontrigger).
+### Persistence Triggers
 
-Persistence configuration files consist of several sections:
+The persistence of an Item's state may be triggered when that Item changes state or when the Item is updated (even if its state did not change).
+Persistence may also be triggered by a time-related event (see Cron Persistence Triggers below).
 
-### Strategies section
+### Strategies
 
-This section allows you to define strategies and to declare a set of default strategies to use for this persistence service.  The syntax is as follows:
+This section allows you to define one or more `Strategies` that will be applied to `Items` (you specifiy the `Items` in the `Items` section).
+This section also defines a default strategy.
+The syntax is as follows:
 
 ```java
 Strategies {
   <strategyName1> : "cronexpression1>"
+  <strategyName2> : "cronexpression2"
+  ...
+  default = everyChange
   ...
 }
 ```
 
-The following strategies are defined internally:
+The `default` parameter assigns a strategy to be used if one is not specified in the `Items` section below.
+The `default` parameter may be omitted from the `Strategies` section, but only if a strategy is provided in each line of the `Items` section.
+
+#### Predefined Strategies
+
+The following strategies are defined internally and may be used in place of `strategyName` above:
 
 - everyChange: persist the Item state whenever its state has changed
 - everyUpdate: persist the Item state whenever its state has been updated, even if it did not change
-- restoreOnStartup: load and initialize the last persisted Item state on openHAB startup (if the Item state is undefined (`UNDEF`)).  See below.
+- restoreOnStartup: load and initialize the last persisted state of the Item on openHAB startup (if the Item state is undefined (`UNDEF`)).  See below.
 
-### Items section
+#### Cron Persistence Triggers
+openHAB uses [Quartz](http://www.quartz-scheduler.org/documentation/quartz-2.1.x/quick-start.html) for time-related cron events.
+In the `Strategies` syntax example above, "cronexpression" is a Quartz cron-like expression.
+See the  [Quartz cron tutorial](http://www.quartz-scheduler.org/documentation/quartz-2.1.x/tutorials/crontrigger) for examples of valid time-related trigger expressions.
```

Comment:
The cron topic is also addressed under rules. There the description might also not be perfect but at least some useful links are given. Imho we should link to there instead of repeating incomplete info here!? http://docs.openhab.org/configuration/rules-dsl.html#time-based-triggers

Reply:
Yes - I looked at the Rules page when I wrote this.  I agree that, to the greatest extent possible, there should be "one source of the truth" rather than scattering information around in different places.  So...

Does this mean that there should be a separate article on Quartz and that both the Rules and Persistence articles should link to it?

Do you know for sure if you can use "midnight" and "noon" in persistence rules?  This is why I did not point to the Items article.  (Also, I did some brief digging but did not find any definite information about whether terms other than those two are supported.  Or...

I can take the quick way out and just point to the Rules article (but there is no sub-link that I can point to that would take them straight to the Time section of the Rules document.  (I could add that as a quick PR.)

Thoughts?

Reply:
Okay - I will change the examples to your heating example.  I agree something that is not updated by a binding is better.

Reply:
I think it would be okay to link to the rules article. I'm not sure what you mean by sub-link. This one should be it?
http://docs.openhab.org/configuration/rules-dsl.html#time-based-triggers

Regarding midnight and all other doubts: Honestly you can imho ignore that for now. There is a totally new scheduler at the verge of being merged to eclipse smarthome. We'll have to touch documentation because of that soon enough.

Reply:
-  Okay - I will simply link to that article.
-  Yep - that is what I meant by the sub-link.  (But I did not see the little chain link symbol on the documentation page in HTML, so I assumed the links had not been added.  I need to go look at the source for the Rules page.)
-  Okay - I was just concerned that someone might dive into the Rules link, see the "midnight" terminology and immediately put it into a persistence definition only to have it blow up (or work - I have no idea whether it is supported in persistence configs or not).  Not a big deal.